tests/actions/ReportTest.js

```diff
@@ -107,4 +107,6 @@ describe('actions/Report', () => {
                 expect(resultAction.loading).toEqual(false);
             });
     });
+
+    // TODO: it('should update pins when togglePinned event is handled via Pusher');
```

Comment:
Added this in case we thought it would be useful to add a test here - Pinning is quite basic compared to report comment / action updating, so I didn't write tests yet - can do if we think it's useful!

Reply:
Yes, let's add a test, thanks!

Reply:
Done! I did a bit of refactoring too to DRY it up a bit